projects/js-packages/components/components/pricing-table/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { __ } from '@wordpress/i18n';
+import classnames from 'classnames';
+import {
+	Fragment,
+	createContext,
+	useContext,
+	Children,
+	cloneElement,
+	PropsWithChildren,
+	ReactElement,
+} from 'react';
+import React, { CSSProperties } from 'react';
+import useBreakpointMatch from '../layout/use-breakpoint-match';
+import Text from '../text';
+import styles from './styles.module.scss';
+import {
+	PricingTableProps,
+	PricingTableColumnProps,
+	PricingTableHeaderProps,
+	PricingTableItemProps,
+} from './types';
+
+const PricingTableContext = createContext( undefined );
+
+export const PricingTableItem: React.FC< PricingTableItemProps > = ( {
+	isIncluded,
+	rowLabel,
+	label = null,
+} ) => {
+	const [ isLg ] = useBreakpointMatch( 'lg' );
+	let defaultLabel = isIncluded ? __( 'Included', 'jetpack' ) : __( 'Not included', 'jetpack' );
+	defaultLabel = isLg ? defaultLabel : rowLabel;
+
+	if ( ! isLg && ! isIncluded ) {
+		return <></>;
+	}
+
+	return (
+		<Text
+			variant="body-small"
+			data-check={ isIncluded }
+			className={ classnames( styles.item, styles.value ) }
+		>
+			{ label || defaultLabel }
+		</Text>
+	);
+};
+
+export const PricingTableHeader: React.FC< PricingTableHeaderProps > = ( { children } ) => (
+	<div className={ styles.header }>{ children }</div>
+);
+
+export const PricingTableColumn: React.FC< PricingTableColumnProps > = ( { children } ) => {
+	const [ isLg ] = useBreakpointMatch( 'lg' );
+	const items = useContext( PricingTableContext );
+	const Wrapper = isLg ? Fragment : 'div';
+	const wrapperProps = ! isLg ? { className: styles.card } : {};
+	let index = 0;
```

Comment:
You can remove it, the second received parameter in `Children.map` is the `index`.

```
Children.map( children, ( child, index ) => ... )
```

Reply:
I've left this in, because I want the `index` of the `PricingTableItem`, not just anything in `PricingTableColumn`. For example, if there's a `PricingTableHeader` in the column, the indexes of the items will be `1` and `2` (after the headers), but the indexes of the related labels will still be `0` and `1`.

That being said, there's probably a better way to do this so I'm very much open to suggestions.